src/panels/lovelace/cards/hui-sensor-card.ts

```diff
@@ -195,46 +245,50 @@ class HuiSensorCard extends EventsMixin(LitElement) {
     return path;
   }
 
-  _midPoint(Ax, Ay, Bx, By) {
-    const Zx = (Ax - Bx) / 2 + Bx;
-    const Zy = (Ay - By) / 2 + By;
-    return [Zx, Zy];
+  private _midPoint(
+    _Ax: number,
+    _Ay: number,
+    _Bx: number,
+    _By: number
+  ): number[] {
+    const _Zx = (_Ax - _Bx) / 2 + _Bx;
+    const _Zy = (_Ay - _By) / 2 + _By;
+    return [_Zx, _Zy];
   }
 
-  async _getHistory() {
+  private async _getHistory(): Promise<string> {
     const endTime = new Date();
     const startTime = new Date();
-    startTime.setHours(endTime.getHours() - this._config.hours_to_show);
-    const stateHistory = await this._fetchRecent(
-      this._config.entity,
-      startTime,
-      endTime
-    );
+    startTime.setHours(endTime.getHours() - this._config!.hours_to_show!);
 
-    if (stateHistory[0].length < 1) return;
+    const stateHistory = await this._fetchRecent(startTime, endTime);
+
+    if (stateHistory[0].length < 1) {
+      return "";
+    }
     const coords = this._coordinates(
       stateHistory[0],
-      this._config.hours_to_show,
+      this._config!.hours_to_show!,
       500,
-      this._config.detail
+      this._config!.detail!
     );
-    this._line = this._getPath(coords);
+    return this._getPath(coords);
   }
 
-  async _fetchRecent(entityId, startTime, endTime) {
+  private async _fetchRecent(startTime: Date, endTime: Date): Promise<{}> {
     let url = "history/period";
-    if (startTime) url += "/" + startTime.toISOString();
-    url += "?filter_entity_id=" + entityId;
-    if (endTime) url += "&end_time=" + endTime.toISOString();
-
-    return await this._hass.callApi("GET", url);
-  }
+    if (startTime) {
+      url += "/" + startTime.toISOString();
+    }
+    url += "?filter_entity_id=" + this._config!.entity;
+    if (endTime) {
+      url += "&end_time=" + endTime.toISOString();
+    }
 
-  getCardSize() {
-    return 3;
+    return this.hass!.callApi("GET", url);
   }
 
-  _style() {
+  private renderStyle(): TemplateResult {
```

Comment:
Can we move this to be right under the `render` function?

Reply:
Balloob Advised its the least import so put it at the end